docs/api/features/manifold.rst

```diff
@@ -106,7 +106,18 @@ the ``f_classif`` score to find the 3 best features in our occupancy dataset.
         ("viz", Manifold(manifold='isomap', target='discrete')),
     ])
 
-    X, y = load_occupancy_data()
```

Comment:
I didn't change this to `X, y = load_occupancy()` since it would return a ValueError. I know @bbengfort is still making changes to `yellowbrick.datasets`, so once that's completed, I'll update all of our code snippets. So for now, I've simply changed it back to our original way to load the sample datasets.